src/frontend/webpack.config.js

```diff
@@ -0,0 +1,100 @@
+const path = require("path");
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const webpack = require("webpack");
+
+require("dotenv").config({ path: "./.env" });
```

Comment:
This enables webpack to pick up env vars from `.env`.

Reply:
Sounds good. This approach, with some added configs, works well: https://github.com/azavea/echo-locator/pull/457/commits/a879f6bdb2a81eab6dc20343709a66451c8a7836